### ISSUE TITLE
idXML/featureXML generated in COMET can't open in TOPPview when N-terminal pyro-Gln is present

### ISSUE THREAD

**Issue by bpalanski** [OpenMS/OpenMS#3427]
I have a minimal workflow that does filtering on a COMET search file (output as .pepxml), via IDFileConverter to PeptideIndexer to IDFilter, and the output is idXML. The idXML file has all of the N-terminal pyro-Glu residues annotated as ".(Gln->pyro-Glu)PEPTIDE". 

When I go to open the idXML (or anything downstream of this, such as a featureXML generated using FeatureFinderIdentification), TOPPview gives the error: Error while loading file: Cannot convert string to amino acid sequence: unexpected character '.' in .(Gln-> pyro-Glu)PEPTIDE.

Any fixes other than manually removing the . ?

**Comment by hroest** [comment]
Which version of OpenMS / TOPPView are you using? I just tried this on 2.3 and I can open a file with the string that you have without problems: 
![screenshot from 2018-05-01 19-05-03](https://user-images.githubusercontent.com/464014/39497752-b51d7766-4d72-11e8-8ef2-ad68f0955f5a.png)

Btw: I simply edited one of the example idXML `BSA1_OMSSA.idXML` and added the string that you have

**Comment by hroest** [comment]
PS: I can confirm that this wont work with OpenMS 2.1 or earlier and I get the same error:

```
==============================================================================
19:09:11 ERROR: Error while loading file:
Cannot convert string to amino acid sequence: unexpected character '.' in: .(Gln->pyro-Glu)PEPTIDE
```

**Comment by bpalanski** [comment]
Thank you. Somehow I was running TOPPView 2.1 even with OpenMS 2.3 installed. Can confirm that the error is resolved in TOPPView 2.3.
